tests/bunit.core.tests/TestServiceProviderTest.cs

```diff
@@ -122,5 +132,60 @@ public void Test020()
 
 			actual.ShouldBe(expected);
 		}
+
+		[Fact(DisplayName = "No registered service returns null")]
+		public void Test021()
+		{
+			using var sut = new TestServiceProvider();
+			var result = sut.GetService(typeof(DummyService));
```

Comment:
```suggestion

			var result = sut.GetService(typeof(DummyService));
```

I like to split the arrange act assert steps by spaces to improve readability of the tests.